src/ServiceBouncer/MainForm.cs

```diff
@@ -8,13 +8,17 @@
 using System.Threading;
 using System.Threading.Tasks;
 using System.Windows.Forms;
+using System.Net.Sockets;
 
 namespace ServiceBouncer
 {
     public partial class MainForm : Form
     {
         private readonly BindingList<ServiceViewModel> services = new SortableBindingList<ServiceViewModel>();
         private bool isActive;
+        private string _machineHostname = "localhost";
+
```

Comment:
There doesn't seem to be a need to have this property and field.
Since both are private, just use the field.
Could you also initialise it in the ctor rather than inline